funnel/templates/index.html.jinja2

```diff
@@ -91,7 +91,17 @@
   <script src="{{ manifest('index.js') }}" type="text/javascript"></script>
   <script type="text/javascript">
     $(function() {
-      window.Hasgeek.homeInit();
+      var tickets;
+      {% if featured_project.features.tickets() -%}
+        tickets = {
+          boxofficeUrl: {{ config['BOXOFFICE_SERVER']|tojson }},
+          widgetElem: "#boxoffice-widget",
+          org: {{ featured_project.boxoffice_data.org|tojson }},
+          itemCollectionId: {{ featured_project.boxoffice_data.item_collection_id|tojson }},
+          itemCollectionTitle: {{ featured_project.title|tojson }}
+        };
+      {%- endif %}
+      window.Hasgeek.homeInit('.markdown', tickets);
```

Comment:
Do we need this when `tickets` is not defined? Couldn't these two lines go inside the if block?

Reply:
yes, `homeInit` has other fns required for homepage